src/functions/frontend/toDelta.m

```diff
@@ -33,8 +34,10 @@
         error('toDelta:toDelta', ['When providing a single argument for toDelta, ',...
               'that input must be a ureal or ultidyn']);
     end
-
+elseif nargin == 2
+    delta_out = normalizeLft(toLft(varargin{1}(erase(rct_obj.Name, 'Normalized')))).delta.deltas{1};
```

Comment:
What's the purpose of normalizing the Delta and removing the "Normalized" qualifier from `rct_obj.Name`?